dracut/30afterburn/afterburn-hostname.service

```diff
@@ -1,14 +1,9 @@
 [Unit]
-Description=Afterburn Hostname
+Description=Afterburn Hostname DEBUG USING IDENTITY ENDPOINT
 # These platforms do not provide the hostname via DHCP
 # options, thus it needs to be fetched from the metadata
 # and statically applied on first-boot.
-ConditionKernelCommandLine=|ignition.platform.id=aliyun
-ConditionKernelCommandLine=|ignition.platform.id=azure
-ConditionKernelCommandLine=|ignition.platform.id=digitalocean
-ConditionKernelCommandLine=|ignition.platform.id=exoscale
-ConditionKernelCommandLine=|ignition.platform.id=ibmcloud
-ConditionKernelCommandLine=|ignition.platform.id=vultr
+ConditionKernelCommandLine=|ignition.platform.id=azurestack
```

Comment:
> there is a messy change in afterburn-hostname.service that breaks other platforms, but I think I should be able to just add them back in....

Yeah, there shouldn't be any issue there at all with just adding this line only to the original service unit.